RecoLocalTracker/SiPixelClusterizer/plugins/gpuClusterChargeCut.h

```diff
@@ -19,105 +19,106 @@ namespace gpuClustering {
       uint32_t const* __restrict__ moduleId,     // module id of each module
       int32_t* __restrict__ clusterId,           // modified: cluster id of each pixel
       uint32_t numElements) {
-    if (blockIdx.x >= moduleStart[0])
```

Comment:
very sorry. code-format has made the few (trivial) changes completely swamped by the new indentation

Reply:
No problem - hiding the whitespace changes in git (`diff -w`) or in GitHub (gear icon --> `Hide whitespace changes`) alleviates the issue.